package.json

```diff
@@ -56,7 +56,7 @@
     "ember-router-generator": "^1.2.3",
     "jquery": "^3.2.1",
     "resolve": "^1.3.3",
-    "rsvp": "^3.6.1",
+    "rsvp": "^4.0.2",
```

Comment:
Can you update this to 4.7.0 now that it is published?

Reply:
done 👍